### ISSUE TITLE
Set focus even if active window is on top

### ISSUE THREAD

**Issue by schrmh** [GaZaTu/x11-emoji-picker#24]
The emoji picker sets focus and I can type starting when I launch it but only when the previously focused window is not on top (I use `wmctrl -r :ACTIVE: -b toggle,above` to toggle that). I would like to have an option to make the emoji-picker take the focus even when the currently active window is on top.

I'm on Arch using Xfce4/xfwm4.

**Comment by GaZaTu** [comment]
ok first thoughts: I could not reproduce this when running `wmctrl -r :ACTIVE: -b toggle,above` on `Konsole` with my Manjaro/KDE machine; the emoji picker was still able to input text and it was infront of `Konsole` aswell.
I'll have to install Xfce4 i guess.

**Comment by schrmh** [comment]
Yeah, in my case the picker does open above but the focus stays at the old window.
But it should work on Xfce4/xfwm4:
I wrote a small app using xlib which launches a window and I can successfully raise and then set focus to the window and it works, no matter whether the previously focused window was on top or not.

**Comment by GaZaTu** [comment]
i'm actually a bit confused because i can't get the emoji picker to work at all with Xfce4 😅
As in, it starts but doesn't input anything regardless of target app and input method.

**Comment by GaZaTu** [comment]
@schrmh what if you try https://github.com/GaZaTu/x11-emoji-picker/actions/runs/2157840101 (the .AppImage whenever it's done) 🤔

**Comment by schrmh** [comment]
So what I noticed.
I had "Prevent focus stealing on" which lead to the case that wmctrl -a did not work.
One might think now that disabling this solves the problem here but without the option the emoji picker doesn't get focus no matter if xfce4-terminal is on top or not... :D

**Comment by schrmh** [comment]
Okay, actually.... When I try it with my local copy which still is at 7650a30914fc4fe0ca332d7674f9206bef408e4e it works as expected when "Prevent focus stealing" is turned off.
